projects/spectator/src/lib/internals.ts

```diff
@@ -290,13 +281,7 @@ export class Spectator<C> {
  */
 export function _getChild<R>(debugElementRoot: DebugElement) {
   return function(directiveOrSelector: Type<R> | DOMSelector | string, options: { read } = { read: undefined }): R {
-    const [child] = _getChildren<R>(debugElementRoot)(directiveOrSelector, options);
-
-    if (!child) {
-      throw new SpectatorDebugElementNotFoundError(`Cannot find a debug element for ${directiveOrSelector}`);
-    }
-
-    return child;
+    return _getChildren<R>(debugElementRoot)(directiveOrSelector, options)[0] || null;
```

Comment:
I think we should at least add console info so that users will know what is missing.

Reply:
But that was before also not the case, right? Because the error was catched?

Reply:
There was `throw error`

Reply:
Oh, sorry, I see. I'll fix it.

Reply:
Oh, well, it is still thrown right? Because it is not catched?